src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx

```diff
@@ -310,15 +310,15 @@ The agent automatically instruments these frameworks and libraries:
     The agent automatically instruments the following app/web servers. To install the Java agent on supported app/web servers, see [Install the Java agent](/docs/agents/java-agent/installation/java-agent-manual-installation).
 
     * ColdFusion 10
-    * Glassfish 3.0 to 5.x
+    * Glassfish 3.0 to 6.x
     * JBoss 7.0 to latest
     * JBoss EAP 6.0 to 7.3
-    * Jetty 7.0.0.M3 to 10.x
+    * Jetty 7.0.0.M3 to 11.x
     * Mule ESB 3.4 to 3.8.x
     * Netty 3.3.0.Alpha1 to 5.0.0.Alpha1
     * Resin 3.1.9 to 4.0.x
     * Spray-can 1.3.1 to latest
-    * Tomcat 7.0.0 to 9.0.x
+    * Tomcat 7.0.0 to 10.0.x
```

Comment:
@tbradellis for some of these server entries, are we sure we want to refer to the currently latest versions (eg: specific versions) instead of saying `to latest`?

Reply:
Ah, yeah agree...I just replaced the 9, but realize now that 10 is the latest, so it should be latest.